microcosm_flask/errors.py

```diff
@@ -147,14 +147,34 @@ def make_json_error(error):
     return dump_response_data(None, response_data, status_code, headers)
 
 
+def make_json_error_with_sentry(error):
+    response = make_json_error(error)
+    try:
+        send_error_to_sentry(error, response)
+    except ImportError:
+        pass
+    return response
+
+
+def send_error_to_sentry(error, response):
+    from sentry_sdk import capture_exception
+    from sentry_sdk import configure_scope
```

Comment:
I understand that those are here to avoid loading all the sentry stuff if we don't need them; but I think it highlights that having a separate component to send exceptions to sentry would make sense.

Reply:
Agreed. I have a rough structure sketched out for a `microcosm_flask` library and this was an element to be included. Hesitant to include this component as part of this PR would rather wait for lib. Want to get the pubsub/flask parts enabled for a service to see what added metadata I need to add to make each function well in sentry.

As development will be even slower once I need to make this it's own library as to see any sort of change to a service I would need to:
- merge to sentry lib
- release that
- update flask/pubsub lib
- release that
- update service
Not exactly conducive so a iterative development cycle 😆

Reply:
Woo `microcosm` monorepo!

(I'll get my coat).